gravity-forms/gw-advanced-merge-tags.php

```diff
@@ -490,6 +490,13 @@ public function handle_field_modifiers( $value, $input_id, $modifier, $field, $r
 					// Example {My Address Field:1.6:abbr}
 					$default_countries = array_flip( GF_Fields::get( 'address' )->get_default_countries() );
 					return rgar( $default_countries, $value );
+				default:
```

Comment:
Can we use a `case` here and then `parse_modifiers` (or from `$modifiers` if it has the value) instead of the additional regex?

Reply:
`$modifier` has the value `selected[0]` or `selected[1]`. We cannot directly add a `case` condition, until we pre-prcocess.